### PR TITLE
Minor Text Fix for the Latex Strait Leotard

### PR DESCRIPTION
**Pull request by Shiranui-Izayoi**
This is a minor fix to the chat room dialog for the Latex Strait Leotard using DestinationCharacterName instead of DestinationCharacter and replacing leotard with AssetName.

This does slightly change the behaviour as, but I feel it is for the better. The main difference is when someone changes the properties a leotard they are wearing it will currently say "her", but if someone else changes it it instead says "Name's". With the new behaviour it will always be "Name's" instead which should clear up uncertainty and also be more consistent.

The other part is that it now uses the assets name directly instead of just leotard so it will be instantly translated into other languages if it is able to. Basically if the asset has a different name in another language,

This PR is reliant on Ellie's PR that updates how Modular items works which is this one [#2892](https://github.com/Ben987/Bondage-College/pull/2892). So this should only be merged after that one has.

### CODE REVIEW

**Comment by Ruilov3** [comment]
just a very minor grammar correction that was present in the original that can be addressed here - `torso items is now hidden` should likely be "torso items _are_ now hidden" due to "items" being plural

File: BondageClub/Screens/Character/Player/Dialog_Player.csv [hunk 1]
```diff
@@ -2129,14 +2129,14 @@ ItemArmsStraitLeotardOptionnp0,,,Hidden,,
...
-ItemArmsStraitLeotardSetcl1,,,DestinationCharacter clothes are now visible when wearing their leotard.,,
-ItemArmsStraitLeotardSetco0,,,DestinationCharacter torso items is now hidden when wearing their leotard.,,
-ItemArmsStraitLeotardSetco1,,,DestinationCharacter torso items is now visible when wearing their leotard.,,
-ItemArmsStraitLeotardSetnp0,,,DestinationCharacter breast items are now hidden when wearing their leotard.,,
-ItemArmsStraitLeotardSetnp1,,,DestinationCharacter breast items are now visible when wearing their leotard.,,
-ItemArmsStraitLeotardSetvp0,,,DestinationCharacter pelvis and vulva items are now hidden when wearing their leotard.,,
-ItemArmsStraitLeotardSetvp1,,,DestinationCharacter pelvis and vulva items are now visible when wearing their leotard.,,
+ItemArmsStraitLeotardSetcl0,,,DestinationCharacterName clothes are now hidden when wearing their AssetName.,,
+ItemArmsStraitLeotardSetcl1,,,DestinationCharacterName clothes are now visible when wearing their AssetName.,,
+ItemArmsStraitLeotardSetco0,,,DestinationCharacterName torso items is now hidden when wearing their AssetName.,,
```
Comment (ace-1331) [inline BondageClub/Screens/Character/Player/Dialog_Player.csv:2134]:
```suggestion
ItemArmsStraitLeotardSetco0,,,DestinationCharacterName torso items are now hidden when wearing their AssetName.,,
```

File: BondageClub/Screens/Character/Player/Dialog_Player.csv [hunk 2]
```diff
@@ -2129,14 +2129,14 @@ ItemArmsStraitLeotardOptionnp0,,,Hidden,,
...
+ItemArmsStraitLeotardSetco1,,,DestinationCharacterName torso items is now visible when wearing their AssetName.,,
```
Comment (ace-1331) [inline BondageClub/Screens/Character/Player/Dialog_Player.csv:2135]:
```suggestion
ItemArmsStraitLeotardSetco1,,,DestinationCharacterName torso items are now visible when wearing their AssetName.,,
```

**Review by ace-1331 — COMMENTED** [review]
Like Rui said, would be worth to correct the mistake at the same time. (Ive suggested them with this comment to make it simpler)
But this is ok to be approved once the dependency is merged.

**Comment by Shiranui-Izayoi** [comment]
Yep I will fix them when I am awake. I thought I had replied guess I was to tired thanks for waiting.

**Comment by Shiranui-Izayoi** [comment]
Ok all done sorry for the wait. I was kinda out of it when I saw the comments. All done now. If there is anything else let me know.

**Comment by Shiranui-Izayoi** [comment]
> 
> 
> Ok all done sorry for the wait. I was kinda out of it when I saw the comments. All done now. If there is anything else let me know.

Also thanks to @Ruilov3 for finding the mistake in the first place. When I was initally making it I must have originally had something that had is as the correct word but then changed it and forgot to rewrite it.

**Comment by Ben987** [comment]
Thanks for the fixes!  Cheers :)

**Pull request merged**